src/test/java/com/academia/library/util/TestDataCreator.java

```diff
@@ -1,29 +1,26 @@
 package com.academia.library.util;
 
 import com.academia.library.dto.AuthorRequestDto;
-import com.academia.library.dto.AuthorResponseDto;
 import com.academia.library.dto.BookRequestDto;
-import com.academia.library.dto.BookResponseDto;
 import com.academia.library.dto.TagRequestDto;
-import com.academia.library.dto.TagResponseDto;
 
 import java.math.BigDecimal;
-import java.time.LocalDateTime;
 
 public class TestDataCreator {
-    public static TagRequestDto TEST_TAG = TagRequestDto.builder()
-            .name("Abstract name")
-            .build();
+    public static TagRequestDto TEST_TAG = createTestTag();
 
     public static AuthorRequestDto TEST_AUTHOR = AuthorRequestDto.builder()
             .firstName("firstName")
             .lastName("lastName")
             .build();
 
-    public static BookRequestDto TEST_BOOK = BookRequestDto.builder()
-            .price(new BigDecimal("10.00"))
-            .title("Abstract name")
-            .createAt(LocalDateTime.now())
-            .updateAt(LocalDateTime.now())
-            .build();
+    public static String TEST_BOOK_TITLE = "Abstract name";
+
+    public static BigDecimal TEST_BOOK_PRICE = new BigDecimal("10.00");
+
```

Comment:
In camelcase should be static final variable only. Please pay attention to basic code style recondition and "Clean Code"